validateFileNames.js

```diff
@@ -67,23 +75,28 @@ if (filesInViolation.length > 0) {
   shelljs.exit(1);
 } else {
   // No, exit with 0
+  console.info('All checked files are camelCased');
   shelljs.exit(0);
 }
 
 function getFiles() {
   // Let's only validate files managed by git
-  const files = shelljs.exec('git ls-files', { silent: true });
-
-  return files.split('\n').filter(file => {
-    // remove empty strings from the array and remove files in ignored paths
-    return file.length !== 0 && !isIgnored(file);
-  });
+  const { code, stdout } = shelljs.exec('git ls-files', { silent: true });
```

Comment:
This is strange. It always was an object for me (even in `deploy.js` of [hollowverse/hollowverse-docker](https://github.com/hollowverse/hollowverse-docker))

![screenshot_20170820_232052](https://user-images.githubusercontent.com/1012761/29498118-62019bde-85fe-11e7-80fe-0d4ea0322b84.png)